src/lib.rs

```diff
@@ -1222,7 +1222,7 @@ impl<K: KVStore + Sync + Send + 'static> Node<K> {
 
 	/// Send a spontaneous, aka. "keysend", payment
 	pub fn send_spontaneous_payment(
-		&self, amount_msat: u64, node_id: PublicKey,
+		&self, amount_msat: u64, node_id: PublicKey, custom_tlvs: Vec<TlvEntry>,
```

Comment:
are the `custom_tlvs` required in order to send a spontaneous payment?

Reply:
Not really required but may be useful. But I see the point, we'll probably take tnull's suggestion to base this change off PR 270; probably once it's merged